src/features/colinks/wizard/CoLinksWizard.tsx

```diff
@@ -136,13 +134,7 @@ export const CoLinksWizard = () => {
   );
 
   const readyData =
-    keyData &&
-    myProfile &&
-    data &&
-    chainId &&
-    account &&
-    address &&
-    checkEthDenverInvitee;
+    keyData && myProfile && data && chainId && address && checkEthDenverInvitee;
```

Comment:
we really need to ditch the EthDenver thing maybe i'll do that in a sep PR

Reply:
yeah, nice to do in separate PR